frontend/packages/console-shared/src/components/badges/Badge.scss

```diff
@@ -5,7 +5,9 @@
   }
 }
 
-.tech-preview {
+span.ocs-preview-badge {
```

Comment:
@rottencandy Why `span`? What was wrong with the suggested change I gave?

The problem here is you're tying these styles to a semantic rendering, and there is nothing tying these styles to a `span` element.

Reply:
Sorry, I read your comment on email and didn't notice that you had suggested a change, so tried coming up with my own solution. Updated now.